app/presenters/user_analytics_presenter.rb

```diff
@@ -226,13 +226,8 @@ def all_time_stats
   end
 
   def cohort_stats
-    cohort_cache_version = 1
-    cohort_cache_key = "user_analytics/#{current_facility.id}/cohort_stats/#{cohort_cache_version}"
-
-    Rails.cache.fetch(cohort_cache_key, expires_in: 7.days) do
-      quarters = Quarter.new(date: Date.current).previous_quarter.downto(3)
-      CohortService.new(region: current_facility, quarters: quarters).call
-    end
+    quarters = Quarter.new(date: Date.current).previous_quarter.downto(3)
+    CohortService.new(region: current_facility, quarters: quarters).call
```

Comment:
@harimohanraj89 The caching is done in the service now, so both callers of this service should benefit from it.

Reply:
Nice. The `RegionReportService` was doing some funky organization-IDs-in-the-cache-key stuff, so I left it untouched. Thanks for consolidating! 👍  I'm guessing we're no longer going to support different users seeing different reports for the same region based on their organizational access?